.mergify.yml

```diff
@@ -0,0 +1,29 @@
+pull_request_rules:
+  - name: automatic approve dependabot pull requests
+    conditions:
+      - author~=dependabot\[bot\]|dependabot-preview\[bot\]
+      - status-success=continuous-integration/travis-ci/pr
+      - status-success=continuous-integration/appveyor/pr
+      - status-success=codecov/project
+      - status-success=codecov/patch
+    actions:
+      review:
+        type: APPROVE
+
+  - name: write issue comment to start codebuild job
+    conditions:
+      - author~=dependabot\[bot\]|dependabot-preview\[bot\]
+      - "#approved-reviews-by>=1"
+    actions:
+      comment:
+        message: go codebuild go
```

Comment:
Can we gate this to when a salt-formula is what is getting updated?

Reply:
are you referring to these? https://github.com/plus3it/watchmaker/tree/develop/src/watchmaker/static/salt/formulas

trying to look for an example of dependabot pr that updates a salt-forumla

Reply:
https://github.com/plus3it/watchmaker/pull/986

Reply:
Proposing the following flow:
* For salt-formula updates PR:
  1) Match title of the title, if begin with `Bump src/watchmaker/static/salt/formulas/`, then create a label called `salt-formula-update`
  2) Create a comment of `go codebuil go` if pr has `salt-formula-update` label 
  3) Auto merge the pr if the codebuild job ran successfully, and the pr has `salt-formula-update` label

* For other dependabots pr, auto-merge after all other CI checks passed.

Reply:
Oh, good idea, yeah, let's use the label applied by dependabot.